charts/pega/templates/_pega-deployment.tpl

```diff
@@ -140,7 +140,30 @@ spec:
 {{- end }}
         - name: {{ template "pegaVolumeCredentials" }}
           mountPath: "/opt/pega/secrets"
-{{ include "pega.health.probes" .root | indent 8 }}
+      # LivenessProbe: indicates whether the container is live, i.e. running.
```

Comment:
There seems to be an indentation issue. "livenessProbe" should be parallel to volumeMounts.
Indenting the newly added lines with another 2 spaces should solve the build failures.